docs/markdown/Reference-tables.md

```diff
@@ -132,6 +132,7 @@ set in the cross file.
 | wasm64              | 64 bit Webassembly       |
 | x86                 | 32 bit x86 processor     |
 | x86_64              | 64 bit x86 processor     |
+| sw_64               | 64 bit sunway processor  |
```

Comment:
Please keep it in alphabetical order

Reply:
Thank you for your kind suggestion, I've fixed it.

Reply:
👍 I suppose it is ok. I have no knowledge about those processors...

Reply:
github has a "squash and merge" button. It's useful for cases like this where we want one commit at the end, but if we wanted two then it would be useless :/